internal/pkg/cli/interfaces.go

```diff
@@ -247,6 +247,18 @@ type wsSvcDirReader interface {
 	CopilotDirPath() (string, error)
 }
 
+type wsJobLister interface {
+	JobNames() ([]string, error)
+}
+
+type wsJobReader interface {
+	wsJobLister
+}
+
+type wsJobDirReader interface {
+	wsJobReader
+}
```

Comment:
Hmm why can't we just have it like
``` go
type wsJobDirReader interface {
	JobNames() ([]string, error)
}
```

Reply:
Haha-- I was actually going to ask the same thing, but figured the extra layers would make sense as we add complexity. But yes, I'll make the change and adjust if complexity is needed later. Thanks!